core/domain/stats_jobs_one_off.py

```diff
@@ -1694,3 +1694,159 @@ def reduce(key, values):
                 yield (key, value)
         else:
             yield (key, sum(int(v) for v in values))
+
+
+class FillExplorationStatsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job to regenerate missing ExplorationStats models and entries
+    for all corresponding states in an exploration.
+    """
+
+    INVALID_EXP_STATS_KEY = 'invalid_exp_stats'
+    INVALID_STATE_STATS_KEY = 'invalid_state_stats'
+    VALID_EXP_STATS_KEY = 'valid_exp_stats'
+    VALID_STATE_STATS_KEY = 'valid_state_stats'
+    V1_EXPLORATION_MODEL = 'v1_exploration_model_encountered'
+    NO_STATS_MODELS_AVAILABLE = 'no_stat_models_available'
```

Comment:
Please use simple sentence fragments as values, for example: `invalid_exp_stats` → `Missing or invalid ExplorationStatsModel`.

Reply:
Done.